native/acter/src/api/news.rs

```diff
@@ -381,12 +401,37 @@ impl NewsEntry {
             .await?
     }
 
-    pub fn comments_count(&self) -> u32 {
-        4
+    pub async fn comments_count(&self) -> Result<u32> {
+        let me = self.clone();
+
+        RUNTIME
+            .spawn(async move {
+                let manager = me.comments().await?;
+                Ok(manager.stats().total_comments_count().to_owned())
+            })
+            .await?
+    }
+
+    pub async fn likes_count(&self) -> Result<u32> {
+        let me = self.clone();
+
+        RUNTIME
+            .spawn(async move {
+                let manager = me.reactions().await?;
+                Ok(manager.stats().total_reaction_count().to_owned())
+            })
+            .await?
     }
 
-    pub fn likes_count(&self) -> u32 {
-        19
+    pub async fn my_like_status(&self) -> Result<bool> {
+        let me = self.clone();
+
+        RUNTIME
+            .spawn(async move {
+                let manager = me.reactions().await?;
+                manager.my_status().await
+            })
+            .await?
```

Comment:
The comment about `comments_count` applies here equally. Move all of that onto the ReactionsManager to avoid loading it twice.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1208/commits/e5e7c6a86780009aeeba43cbbfa5d677891a4fd9